docs/examples/general/data_loading/webdataset.ipynb

```diff
@@ -0,0 +1,514 @@
+{
```

Comment:
<ul><li>I'd add an explanation that this function is used to configure an external source node (link to documentation of external source) to provide data from a WebDataset instance.</li><li>I'd rather not call the files shards, since it might confuse readers with the concept of DALI shards.</li><li>"By default this is one extension fetching the data from all image formats supported by the" -&gt; "By default, all image format extensions supported by WebDataset are used"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>then should I call them sth like "webdataset shards"? I mean the issue is that this is what they call it in the webdataset module documentation</p>

Reply:
I'd call it simply files in this context